pages/linux/strace.md

```diff
@@ -3,19 +3,19 @@
 > Troubleshooting tool for tracing system calls.
 > More information: <https://manned.org/strace>.
 
-- Start tracing a specific process by its PID:
+- Start tracing a specific [p]rocess by its PID:
 
 `strace -p {{pid}}`
 
-- Trace a process and filter output by system call:
+- Trace a [p]rocess and filt[e]r output by system call:
 
 `strace -p {{pid}} -e {{system_call_name}}`
```

Comment:
```suggestion
`strace -p {{pid}} -e {{system_call1, system_call2,...}}`
```

Reply:
There should be no space between `,` and `system_call2`

Reply:
Oops, that space was accidental, fixed